facebookresearch/multimodal
# Add model and architecture for Omnivore model

Summary:
[Omnivore](https://arxiv.org/abs/2201.08377) is a vision multimodal model that able to classify RGB image, video, and depth image (RGBD) with a shared trunk parameters.

With this PR, we aim to have the model and architecture class for Omnivore that able to load converted pretrained weight from original author: https://github.com/facebookresearch/omnivore and produce same result.

Test plan:
- Add notebook in `examples/omnivore/LoadOriginalPretrainedWeightAndCompare.ipynb` that load the original omnivore pretrained weight in torchhub and make sure the output is the same
- Add unittest
